src/layers/attention.rs

```diff
@@ -52,16 +52,143 @@ pub enum QkvMode {
     MergedSplitAfter(QkvSplit),
 }
 
+/// ALiBi: Linear biases for attention [Press et al., 2022](https://arxiv.org/abs/2108.12409).
```

Comment:
Question: do we want hyperlinks in the inline descriptions? Maybe they distract from glossing an overview page?

<img width="690" alt="Screenshot 2024-01-15 at 08 35 49" src="https://github.com/oxidized-transformers/oxidized-transformers/assets/49398/e9bda485-ca7a-4bc3-84a1-0823b03d1d1d">

If not, I propose:

```suggestion
/// Linear biases for attention (ALiBi)
```

Reply:
I'd like to keep the links to the papers in some way, if possible. Maybe we can stick to the format you're currently using in your transformer layer PR (`See XZY et al. (2001)`)?

Reply:
Yeah, I think having a link in the detailed description is great to have.